web/src/billing/upgrade.ts

```diff
@@ -45,6 +45,9 @@ export const initialize = (): void => {
             () => {
                 $("#org-upgrade-button-text").show();
                 $("#org-upgrade-button .upgrade-button-loader").hide();
+                const $error_box = $("#autopay-error");
+                const error_text = $error_box.text();
+                $error_box.text(`${error_text} Please fix this issue or use a different card.`);
```

Comment:
What clears the `$error_box`? I feel like this is going to accumulate copies of this ...

Reply:
```js
if (xhr.responseJSON?.msg) {
    $(form_error).show().text(xhr.responseJSON.msg);
}
``

Error text was being reset by this call in helpers.ts. But regardless, the change to clear it and hide seems to good to me as a precaution.

Reply:
I see. I think given that, the right solution here is probably to add an `error_text_suffix` parameter to `create_ajax_request` so that all the code for the error message lives there, rather than having it split up.

Reply:
(Feel free to do this as a follow-up; it'd be a very minor cleanup that's not timely at all, but also probably easy to tack onto something else you're doing on these pages)